TorchSharp/Raw/Vector.tt

```diff
@@ -0,0 +1,153 @@
+﻿<#@ Template Language="C#" #>
+<#@ Assembly Name="System.Core" #>
+<#@ Output Extension=".generated.cs" #>
+<#@ Include File="Types.ttinclude" #>
+
+using System;
+using System.Runtime.InteropServices;
+
+namespace TorchSharp.Raw {
+
+    internal static class Vector {
+
+<#
+foreach (var type in TorchTypeDef.Types) {
+#>
+        // Fill n elements of vector x with the value c.
+        //
+        // Corresponds to the following TH declaration:
+        //
+        // TH_API void THVector_(fill)(scalar_t *x, const scalar_t c, const ptrdiff_t n);
+        [DllImport ("caffe2")]
+        internal extern static void TH<#=type.Name#>Vector_fill(
+            <#=type.Ptr#> x,

```

Comment:
Can we have the parameters in one line? I see that this is what you are doing for the other functions.